serverless.yml

```diff
@@ -11,7 +11,13 @@ provider:
   stage: ${env:ENVIRONMENT}
   timeout: 180 # 3 minutes
   memorySize: 128 # MB
+  iamRoleStatements:
+    - Effect: 'Allow'
+      Action:
+        - 'secretsmanager:GetSecretValue'
+      Resource: 'arn:aws:secretsmanager:${self:provider.region}:${self:provider.environment.AWS_ACCOUNT_ID}:*'
   environment:
+    AWS_ACCOUNT_ID: ${env:AWS_ACCOUNT_ID}
```

Comment:
it's not needed in the lambda process (adding it here it will be available when process.env), you only need it for the serverless process (I mean in CI, so only on github actions VM)

Reply:
I'll remove it in a new PR. Thanks!